operations/mimir-mixin/dashboards/writes.libsonnet

```diff
@@ -137,6 +137,44 @@ local filename = 'mimir-writes.json';
         )
       )
     )
+    .addRowIf(
+      $._config.forwarding_enabled,
+      $.row('Distributor Forwarding')
+      .addPanel(
+        $.panel('Requests / sec') +
+        $.successFailurePanel(
+          |||
+            sum(  
+              rate(
+                cortex_distributor_forward_requests_total{%(distributorMatcher)s}[$__rate_interval]
+              )
+              - on (namespace, pod)
+              rate(
+                cortex_distributor_forward_errors_total{%(distributorMatcher)s}[$__rate_interval]
+              )
+            )
+          ||| % [
+            $.jobMatcher($._config.job_names.distributor),
+            $.jobMatcher($._config.job_names.distributor),
+          ],
+          'sum(rate(cortex_distributor_forward_errors_total{%(distributorMatcher)s}[$__rate_interval]))' % $.jobMatcher($._config.job_names.distributor)
```

Comment:
It would be nice to preserve `status_code`.

Reply:
Is this what you had in mind? 1b3cf584ef

optimally we would just use the `$.qpsPanel()` like the other first panels of the rows above and below this one, but for that we're missing a single metric which contains the rates per status code. Maybe we should refactor the forwarding metrics to have a single metric with all the status_code rates, but that's out of the scope of this PR.

Reply:
Yes, `sum by (status_code)` is enough. Thanks!